move-vm-backend/tests/move_vm.rs

```diff
@@ -2,6 +2,7 @@ use crate::mock::StorageMock;
 use move_core_types::account_address::AccountAddress;
 use move_core_types::identifier::Identifier;
 use move_core_types::language_storage::StructTag;
+use move_core_types::language_storage::CORE_CODE_ADDRESS as ADDR_STD;
```

Comment:
I hope the alias makes sense here

Reply:
If we're putting there only standard library, yes.